cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -1136,3 +1138,41 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
         view.fields("network").read() == network and
         view.fields("number_disk").read() == additional_disks
     )
+
+
+@pytest.mark.meta(automates=[1559030])
+@pytest.mark.tier(2)
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data", [DatastoreImport("bz_1558926.zip", "bz_1558926", None)], ids=["domain"],)
+@pytest.mark.parametrize("file_name", ["bz_1559030.yml"], ids=["dialog"])
+def test_save_dynamic_multi_drop_down_dialog(appliance, import_datastore, import_dialog,
+                                             import_data):
+    """
+    Bugzilla:
+        1559030
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        testtype: functional
+        initialEstimate: 1/4h
+        startsin: 5.10
+    """
+    sd, ele_label = import_dialog
+    navigate_to(sd, "Edit")
+
+    # update dialog element
+    view = appliance.browser.create_view(EditElementView)
+    view.element.edit_element(ele_label)
+    view.options.click()
+
+    # enable multi select button
+    view.options.multi_select.fill("Yes")
+    view.ele_save_button.click()
+    view.save_button.click()
+
+    # no error should be displayed
+    view = sd.create_view(DetailsDialogView)
+    view.wait_displayed()
+    view.flash.assert_success_message(f'{sd.label} was saved')
```

Comment:
I think: you can add ```wait``` in create_view(DetailsDialogView). No need of   ```view.wait_displayed()```.

Reply:
there is some issue with `wait` . `view.wait_displayed()` would be nice here.

Reply:
can you please point that issue here? Is GH issue created for it?

Reply:
not sure the issue but i heard that and there is one PR of Parthvi https://github.com/ManageIQ/integration_tests/pull/9978/files#diff-1af1dca8d2ad3e8627bac286076744ebR491

Reply:
@valaparthvi any issue you know related to **wait**?

Reply:
I don't think there is any problem with create_view's `wait`. That problem was with FlashMessages widget. I think the widget stopped supporting `wait` argument.

Reply:
Thanks @valaparthvi @niyazRedhat